en/node-operators-guide/vrf.md

```diff
@@ -118,14 +118,58 @@ Errors and warnings emitted by the **Orakl Network VRF** can be [sent to Slack c
 
 Before launching the VRF solution, the **Orakl Network API** has to be accessible from the **Orakl Network VRF** to load VRF keys, and listener settings.
 
-After the **Orakl Network API** is healthy, VRF microservices (listener, worker, reporter) can be launched in an arbitrary order. Microservices communicate with each other through the BullMQ - job queue.
+After the **Orakl Network API** is healthy, launch the VRF service, which consists of listener, worker, and reporter microservices, with the command below. Microservices communicate with each other through the BullMQ - job queue.
+
+```sh
+yarn start:core:vrf
+```
+
+For development purposes, the service can be run with `nodemon`, which automatically restarts all the microservices when file changes are detected, using the following command:
+
+```sh
+yarn dev:core:vrf
+```
+
+It's also possible to run the microservices separately in any arbitrary order:
 
 ```sh
 yarn start:listener:vrf
 yarn start:worker:vrf
 yarn start:reporter:vrf
 ```
 
+## Quick launch with Docker
+
+From [orakl]() repository's root, run the following command to build all images:
+
+```bash
+docker-compose -f docker-compose.local-core.yaml build
+```
+
+Set wallet credentials, `ADDRESS` and `PRIVATE_KEY` values, in the [.core-cli-contracts.env](https://github.com/Bisonai/orakl/blob/master/dockerfiles/local-vrf-rr/envs/.core-cli-contracts.env) file. Keep in mind that the default chain is `localhost`. If you'd like to change it to either `baobab` or `cypress`, update `CHAIN` and `PROVIDER_URL` values in the earlier mentioned `.env` file. Note that if the chain is not `localhost`, `Coordinator` and `Prepayment` contracts won't be deployed. Instead, Bisonai's already deployed contract addresses ([VRF](https://github.com/Bisonai/vrf-consumer/blob/master/hardhat.config.ts), [RR](https://github.com/Bisonai/request-response-consumer/blob/master/hardhat.config.ts)) will be used. After setting the appropriate `.env` values, run the following command to start the VRF service:
```

Comment:
how about using permalink which leads to the address line
https://github.com/Bisonai/vrf-consumer/blob/9b06ddd10b472d821878c9994463856664387c85/hardhat.config.ts#L60-L68

and I think you just have to specify vrf link here since it is vrf guide